### PR TITLE
refactor: add comment for `Session` and `Session::close`

### PR DESCRIPTION
**Pull request by wyfo**

### CODE REVIEW

**Review by Mallets — APPROVED** [review]

**Comment by Mallets** [comment]
@wyfo LGTM. However CI is not happy.

**Comment by wyfo** [comment]
Indeed, I'm fixing it.

File: zenoh/src/api/session.rs [hunk 1]
```diff
@@ -410,8 +410,23 @@ impl fmt::Debug for SessionInner {
     }
 }
 
-/// A zenoh session.
+/// The entrypoint of the zenoh API.
 ///
+/// Zenoh session is instantiated using [`zenoh::open`], and can be used to declares various
```
Comment (Mallets) [inline zenoh/src/api/session.rs:415]:
```suggestion
/// Zenoh session is instantiated using [`zenoh::open`], and can be used to declare various
```

File: zenoh/src/api/session.rs [hunk 2]
```diff
@@ -410,8 +410,23 @@ impl fmt::Debug for SessionInner {
...
+/// entities like publishers, subscribers, or querybables, as well as making queries.
```
Comment (Mallets) [inline zenoh/src/api/session.rs:416]:
```suggestion
/// entities like publishers, subscribers, or querybables, as well as issuing queries.
```

**Review by Mallets — APPROVED** [review]

**Pull request merged**